documentation/server/discord/bot.md

```diff
@@ -6,49 +6,49 @@ description: Learn more about the Discord Bot
 
 ## Introduction
 
-Integrate an Discord Bot to your gamemode can have some benefits for you. Creating a whitelist, use the awesome
-permission layer from discord for ingame actions, read some server stats, kick players and so on. The list can be go
+Integrate a Discord Bot to your gamemode can have some benefits for you. Creating a whitelist, use the awesome
+permission layer from Discord itself for in-game actions, read some server stats, kick players and so on. The list can go
 further.
 
-{% hint style="info" %} Creating your own bot can not be simpler as this. {% endhint %}
+{% hint style="info" %} Creating your own bot cannot be simpler as this. {% endhint %}
 
 ## Setup the Bot
 
 ### Step 1
 
 Go to [https://discord.com/developers/applications](https://discord.com/developers/applications) and create an account
-and application it not already exists
+and application if not already existent.
```

Comment:
`and application if not already existent.` to `and application if you do not already have on.`